init/atboot.pl

```diff
@@ -117,8 +119,22 @@
 	$perl_path = &get_perl_path();
 	&enable_at_boot($product, $ucproduct, $perl_path." ".&quote_path("$root_directory/miniserv.pl")." ".&quote_path("$config_directory/miniserv.conf"));
 	}
-elsif ($init_mode eq "rc" || $init_mode eq "upstart" ||
-       $init_mode eq "systemd") {
+elsif ($init_mode eq "systemd") {
```

Comment:
What's going to happen when atboot.pl is run a second time when Webmin is upgraded? Will we end up with an init script and a systemd script?

Reply:
> Will we end up with an init script and a systemd script?

Not for **.rpm** and not for **.deb** packages though. But I missed the fact it may be install from source. What do we do for this? Should we also try to remove init script?

Reply:
Actually, even if installed from source there would be no _init script_ left on `systemd` systems as there is no one installed. It was only done by RPM and DEB and those are auto cleaned, right?

Reply:
Oh, we don't need to support upgrading from source to RPM / DEB. However, we do need to support upgrading from an older RPM that included the init script to an RPM with these changes.

Reply:
Yes, when I meant an upgrade, I meant upgrading when installed from source.

Reply:
Ok, great